### PR TITLE
Support Rails 6

### LINKED ISSUE #91: Support for Active Record 6

### PR DESCRIPTION
**Pull request by tjwallace**
Closes #91.

### CODE REVIEW

**Comment by tjwallace** [comment]
Any idea how I can get TravisCI to run this branch?

**Comment by tjwallace** [comment]
@jesjos let me know what you think of the TravisCI build matrix changes - 11 + 2 builds is quite a lot, maybe we should set the ruby version to only `2.6.2` and be done with it?

File: .travis.yml [hunk 1]
```diff
@@ -4,19 +4,27 @@ language: ruby
...
 gemfile:
+- Gemfile.rails-6-0
 - Gemfile.rails-5-2
 - Gemfile.rails-5-1
 - Gemfile.rails-5-0
 - Gemfile.rails-master
 rvm:
 - 2.4.1
+- 2.5.3
+- 2.6.2
```
Comment (olleolleolle) [inline .travis.yml]: Thanks for extending the matrix!

Could you amend the file to use these?

- 2.4.6
- 2.5.5
- 2.6.3

Comment (tjwallace) [reply .travis.yml]: Yup! Done in 67f12ef

**Comment by tjwallace** [comment]
No problem @olleolleolle!

Do you plan on creating a new release (or beta release) soon?

**Comment by olleolleolle** [comment]
@tjwallace How about right now?

**Comment by tjwallace** [comment]
@olleolleolle that sounds great, thanks!

**Comment by olleolleolle** [comment]
OK, now there's a tag, and there's a GitHub Release, so RubyGems' release will fall out of that as a publish hook.

**Pull request merged**